src/styles/components/settings/settings.css

```diff
@@ -18,20 +18,33 @@
     font-weight: bold;
 }
 
-/*TODO: Add a11y back! */
 .settingsItem__select {
     position: relative;
     font-size: 1.2em;
+    text-align: right;
```

Comment:
Works the same way as rtl for me, but maybe that's an OS specific thing, need to be checked